internal/coding/quotedprint_test.go

```diff
@@ -152,6 +152,40 @@ func TestQPCleanerLineBreakBufferFull(t *testing.T) {
 	}
 }
 
+func TestQPCleanerEqualSignOverflow(t *testing.T) {
+	// Test processing of =3D quoted printable token being at the end of line
+	// in combination with almost full buffer.
+	// qp cleaner should add =\r\n and then output =3D without buffer overflow
+
+	input := append(bytes.Repeat([]byte("abc"), 341), []byte("=3D")...)
+	inbuf := bytes.NewBuffer(input)
+	qp := coding.NewQPCleaner(inbuf)
+
+	output := make([]byte, 1024)
+	n, err := qp.Read(output)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if n != 1024 {
+		t.Errorf("Unexpected result length: %d", n)
```

Comment:
Can we add std Go "got: %v, wanted: %v" syntax to all four of these failures, they will be easily confused.

Reply:
Ok, done. I did not know about such std, personally I use `require` package.

Reply:
Thanks!

Not super important, but if you look through the go std lib, you will see this pattern over and over:

	if got, want := a.Get(&v1), 1; got != want {
		panic(fmt.Sprintf("Get() == %d, want %d", got, want))
	}

Basically they almost always use `got` and `want`, and got is placed before want in both the equality test and the failure output text.

We did introduce stretchr/testify back into enmime recently.  I wouldn't want to add `require` as well, but feel free to use testify assertions if you prefer those to vanilla go.